data/lang/module-soldout/en.json

```diff
@@ -0,0 +1,30 @@
+{
+  "AMOUNT": {
+    "description": "The amount (for now in tonne) of goods person wants to buy",
+    "message": "I'll buy anything you have, up to the full {amount}t that I need."
+  },
+  "MESSAGE": {
+    "description": "",
+    "message": "I'm {name}. I need {commodity} urgently, and I'll pay twice the market price ({price})"
+  },
+  "NOT_IN_STOCK": {
+    "description": "Regarding selling more than the player has",
+    "message": "Not enough in stock"
```

Comment:
I'd recommend adding a `{commodity}` in here, as well as adding punctuation to the end.